### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by sfc-gh-halu**
Explicit instructions on installing the wheel file.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,6 +9,11 @@ and [CLI](https://www.mlflow.org/docs/latest/cli.html#mlflow-deployments) for ML
 ## Usage
 ### Prerequisite
 * The plugin relies on local anaconda installation to check if model dependencies could be satisfied. Suggest install [Miniconda](https://docs.conda.io/en/latest/miniconda.html) for minimum dependencies.
+
+### Installation 
+Please find latest release version [here](https://github.com/Snowflake-Labs/mlflow-snowflake/releases) to download latest `wheel`.
```
Comment (sfc-gh-sdas) [inline README.md:14]: can you use relative path?

https://github.blog/2013-01-31-relative-links-in-markup-files/

File: README.md [hunk 2]
```diff
@@ -9,6 +9,11 @@ and [CLI](https://www.mlflow.org/docs/latest/cli.html#mlflow-deployments) for ML
 ## Usage
 ### Prerequisite
```
Comment (sfc-gh-sdas) [inline README.md:10]: can we add current support matrix? Can we note SKL/XGB are the only supported models?

Comment (sfc-gh-halu) [reply README.md:10]: It's mentioned explicitly in the `limitations` section already.

File: README.md (hunk 1 above)
Comment (sfc-gh-halu) [reply README.md:14]: From link, looks like relative link applies to main branch source files only?

Comment (sfc-gh-sdas) [reply README.md:14]: when you move code from one repo to another (which is possible), relative helps. But I am not sure if relative links work on releases. So LGTM, but please try if you can.

**Review by sfc-gh-sdas — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (sfc-gh-halu) [reply README.md:14]: Tried `/releases` relative path but did not got success.

**Pull request merged**